media/css/firefox/whatsnew/whatsnew-95-en.scss

```diff
@@ -10,7 +10,8 @@ $image-path: '/media/protocol/img';
 @import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 @import '~@mozilla-protocol/core/protocol/css/templates/multi-column';
-@import 'includes/header';
+@import 'includes/base';
+@import 'includes/dark-mode';
```

Comment:
Not a blocker, but I notice most of the pages get the default Firefox footer styles (hover states and bg color are different). Is there a reason why only WNP 95 gets the `base` and `dark-mode` imports?

I'm just wondering if we can simplify anything, so all templates get the same footer styles going forward?

Reply:
I thought I'd leave existing styles and set up the base/dark-mode imports for future pages (with an example of use on fx95). But that was mainly because the PR was getting big. It should be fairly straightforward to make these updates across all remaining WNP templates. I think it's worth doing - thanks for the review!